### PR TITLE
SkyScanner

### PR DESCRIPTION
**Pull request by Carsblox**
## Describe your project in 1-3 sentences

I am creating a flight tracking node named SkyScanner

## Hack Club Slack handle

Mostafa Saleh

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Review by bellesea — CHANGES_REQUESTED** [review]
@Carsblox this is sooo awesome! i'm wondering, are you following any sort of guide for this? and, what made you decide to choose this board?

thank you sm!

**Comment by alialiwa2005** [comment]
> @Carsblox this is sooo awesome! i'm wondering, are you following any sort of guide for this? and, what made you decide to choose this board?
> 
> thank you sm!

Hello, @bellesea! I've recommended the Libre Computer LePotato board to @Carsblox, since his initial pick, the Raspberry Pi 4, is currently facing a supply shortage.

I'm also using similar Libre Computer boards as substitutes for my Winter of Making project, which initially had Raspberry Pis on the materials list. 

After thorough research, we chose to give Libre Computer boards a try because they have fairly positive reviews as a Raspberry Pi alternative, follow an identical form factor as a modern Raspberry Pi (they can be screwed into common standard Pi 3 and Pi 4 mounts), and are affordable when compared to other Raspberry Pi replacements, especially those that are easily available at this time.

Additionally, the LePotato is the "flagship" board of Libre Computer, and is also the best-selling and most affordable model.

**Comment by bellesea** [comment]
> > @Carsblox this is sooo awesome! i'm wondering, are you following any sort of guide for this? and, what made you decide to choose this board?
> > thank you sm!
> 
> Hello, @bellesea! I've recommended the Libre Computer LePotato board to @Carsblox, since his initial pick, the Raspberry Pi 4, is currently facing a supply shortage.
> 
> I'm also using similar Libre Computer boards as substitutes for my Winter of Making project, which initially had Raspberry Pis on the materials list.
> 
> After thorough research, we chose to give Libre Computer boards a try because they have fairly positive reviews as a Raspberry Pi alternative, follow an identical form factor as a modern Raspberry Pi (they can be screwed into common standard Pi 3 and Pi 4 mounts), and are affordable when compared to other Raspberry Pi replacements, especially those that are easily available at this time.
> 
> Additionally, the LePotato is the "flagship" board of Libre Computer, and is also the best-selling and most affordable model.

that's awesome!! thanks sm for the context :)

**Review by bellesea — CHANGES_REQUESTED** [review]
@Carsblox that's awesome!! could you please add the guide it into the file as a source/reference for your project?

also, do you need an SD card? and, how do you plan on connecting to the libre board?

and, where do you plan on showcasing the data that you collect, would it be a web app of sorts? 

btw, what's the role of the case/enclosure?

thank you :)

**Comment by Carsblox** [comment]
> @Carsblox that's awesome!! could you please add the guide it into the file as a source/reference for your project?
> 
> also, do you need an SD card? and, how do you plan on connecting to the libre board?
> 
> and, where do you plan on showcasing the data that you collect, would it be a web app of sorts?
> 
> btw, what's the role of the case/enclosure?
> 
> thank you :)

I added the SD card to the list of materials. 
When initially setting up I plan on connecting to the board with a USB mouse and keyboard. 
The data colleceted will be shown on a display or monitor 
The role of the case is to keep all the parts together and to also provide protection to the components such as the board which is fragile.

**Review by bellesea — CHANGES_REQUESTED** [review]
@Carsblox thanks for pushing the changes! hmm rn it seems like the only components are the board, the dongle, and the antenna? a slightly modified standard case for the board could probably work?

i'm a little hesitant to approve a budget where >80% of it goes towards a non-essential part of the project? would love to know what you think!

thanks :)

File: Carsblox.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+name: "@Carsblox"
+project: "SkyScanner"
+---
+
+# Project Name
```
Comment (bellesea) [inline Carsblox.md:6]:
```suggestion
# SkyScanner
```

File: Carsblox.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+SkyScanner
```
Comment (bellesea) [inline Carsblox.md:7]:
```suggestion
```

**Review by bellesea — APPROVED** [review]
@Carsblox thanks for sharing this project idea, it's super cool and can't wait to see you work on this!!

an awesome member of @hackclub/bank-ops will be reaching out soon with next steps!

**Review by hackliv — APPROVED** [review]
Hey @Carsblox this sounds so cool!! Ali's been sharing a bit about this with me too. :) I'm dming you now on Slack about getting set up on Hack Club Bank with the grant! 🎉

**Pull request merged**